nifi-extension-bundles/nifi-graph-bundle/nifi-neo4j-cypher-service/src/main/java/org/apache/nifi/graph/Neo4JCypherClientService.java

```diff
@@ -69,26 +69,23 @@ public class Neo4JCypherClientService extends AbstractControllerService implemen
             .build();
 
     public static final PropertyDescriptor USERNAME = new PropertyDescriptor.Builder()
-            .name("neo4j-username")
-            .displayName("Username")
+            .name("Username")
             .description("Username for accessing Neo4J")
             .required(true)
             .expressionLanguageSupported(ExpressionLanguageScope.ENVIRONMENT)
             .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
             .build();
 
     public static final PropertyDescriptor PASSWORD = new PropertyDescriptor.Builder()
-            .name("neo4j-password")
-            .displayName("Password")
+            .name("Password")
             .description("Password for Neo4J user. A dummy non-blank password is required even if it disabled on the server.")
             .required(true)
             .sensitive(true)
             .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
             .build();
 
     public static final PropertyDescriptor CONNECTION_TIMEOUT = new PropertyDescriptor.Builder()
-            .name("neo4j-max-connection-time-out")
-            .displayName("Neo4J Max Connection Time Out (seconds)")
+            .name("Neo4J Max Connection Time Out (seconds)")
```

Comment:
Recommend shortening the name following other components:
```suggestion
            .name("Connection Timeout")
```